src/components/TeamBuilderPage/TeamBuilderPage.jsx

```diff
@@ -0,0 +1,179 @@
+import React from "react";
+import ReactTooltip from 'react-tooltip';
+import commonStyles from '../../common.module.css';
+import styles from './TeamBuilderPage.module.css';
+import heroes from 'underlordsconstants/build/underlords_heroes.json';
+import alliances from 'underlordsconstants/build/underlords_alliances.json';
+import { GetHeroImage } from "../../utils";
+import { underlordsLoc } from '../Localization/Localization';
+import { strings } from './../Localization/Localization';
+
+const Synergy = ({synergy, count, level, levelUnitCount}) => {
```

Comment:
Add a TODO to unify this logic with https://github.com/odota/underlords-web/blob/master/src/components/AllianceCard/AllianceCard.tsx